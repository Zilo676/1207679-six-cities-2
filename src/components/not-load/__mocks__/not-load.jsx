import React from "react";

const NotLoad = () => {
  return <></>;
};

export {NotLoad};
export default NotLoad;
