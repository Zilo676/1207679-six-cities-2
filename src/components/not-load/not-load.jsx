import React from 'react';

const NotLoad = () => {
  return (
    <div className="property__mark">
      <span>NOT LOADING</span>
    </div>
  );
};

export {NotLoad};
