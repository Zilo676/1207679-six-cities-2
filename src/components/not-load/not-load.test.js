import React from 'react';
import renderer from 'react-test-renderer';

import {NotLoad} from './not-load.jsx';

it(`Correctly rendered`, () => {
  const tree = renderer.create(<NotLoad />).toJSON();
  expect(tree).toMatchSnapshot();
});
