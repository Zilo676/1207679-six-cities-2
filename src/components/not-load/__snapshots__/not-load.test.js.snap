// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly rendered 1`] = `
<div
  className="property__mark"
>
  <span>
    NOT LOADING
  </span>
</div>
`;
